lib/UtapiClient.js

```diff
@@ -971,6 +971,28 @@ class UtapiClient {
             return undefined;
         });
     }
+
+    /**
+     * Get storage used by bucket/account/user/service
+     * @param {object} params - params for the metrics
+     * @param {string} [params.bucket] - (optional) bucket name
+     * @param {string} [params.accountId] - (optional) account canonical ID
+     * @param {string} [params.userId] - (optional) account ID
+     * @param {string} [params.level] - (required) level of granularity
+     * @param {string} [params.service] - (required) service name (s3 for ex.)
+     * @param {object} log - Werelogs request logger
+     * @param {callback} callback - callback to call
+     * @return {undefined}
+     */
+    getStorageUtilized(params, log, callback) {
```

Comment:
The function passes in a `log` param but it's not used
Should there be a log statement before the `InternalError` so we don't mask the error completely?